mainapp/admin.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib import admin
+
+from .models import Profile, User
+
+
+class UserAdmin(admin.ModelAdmin):
```

Comment:
add proper user model fields in admin. Make different sections as I sent you previously.